pallets/compliance-manager/src/lib.rs

```diff
@@ -571,6 +571,19 @@ impl<T: Trait> Module<T> {
             .collect::<Vec<_>>()
     }
 
+    /// It fetches the `ConfidentialScopeClaim` of users `id` for the given ticker.
+    /// Note that this vector could be 0 or 1 items.
+    fn fetch_confidential_claims(id: IdentityId, ticker: &Ticker) -> Vec<Claim> {
+        let claim_type = ClaimType::InvestorZKProof;
+        let asset_scope = IdentityId::try_from(ticker.as_slice())
+            .expect("Ticker len should be less than asset scope");
```

Comment:
We shouldn't have panics in the runtime. This condition should never be triggered anyway so maybe use something like `unwrap_or_default` to just be safer?

Reply:
@max